src/app/public/modules/flyout/flyout-media-query.service.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  Injectable
+} from '@angular/core';
+
+import {
+  BehaviorSubject
+} from 'rxjs/BehaviorSubject';
+
+import {
+  Subscription
+} from 'rxjs/Subscription';
+
+import {
+  SkyMediaBreakpoints
+} from '@skyux/core';
+
+import {
+  SkyMediaQueryListener
+} from '@skyux/core/modules/media-query/media-query-listener';
+
+import {
+  SkyFlyoutAdapterService
+} from './flyout-adapter.service';
+
+@Injectable()
+export class SkyFlyoutMediaQueryService {
+
+  public get current(): SkyMediaBreakpoints {
+    return this._current;
+  }
+
+  private currentSubject = new BehaviorSubject<SkyMediaBreakpoints>(this.current);
+
+  private _current = SkyMediaBreakpoints.xs;
+
+  constructor(
+    private adapterService: SkyFlyoutAdapterService
+  ) {
+    this.currentSubject.next(this._current);
+  }
+
+  public subscribe(listener: SkyMediaQueryListener): Subscription {
+    return this.currentSubject.subscribe({
+      next: (breakpoints: SkyMediaBreakpoints) => {
+        listener(breakpoints);
+      }
+    });
+  }
+
+  public setBreakpoint(width: number) {
```

Comment:
I found it difficult to figure out what this method was doing, initially. Looking through the code, it appears to be finding the nearest (or lowest?) breakpoint based on a given pixel width? If that's correct, it might be worth finding a method name that communicates that a bit more clearly.

Reply:
Also needs output type ☝️

Reply:
Output type is done.

Reply:
You're correct in that the method sets the appropriate breakpiont based on the width that is passed in (due to the fact that we don't have listeners like the "real" version does). 

What would you think about `setBreakpointForWidth` @Blackbaud-SteveBrush? I don't think I want to say `setBreakpointForFlyoutWidth` because if we ever do implement an interface for these I would like the method names to be more generic.

Reply:
That works!

Reply:
Done